zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryRequestMapper.java

```diff
@@ -675,6 +702,22 @@ private static List<String> applyTenantSortField(
     return validationErrors;
   }
 
+  private static List<String> applyMappingSortField(
+      final String field, final MappingSort.Builder builder) {
+    final List<String> validationErrors = new ArrayList<>();
+    if (field == null) {
+      validationErrors.add(ERROR_SORT_FIELD_MUST_NOT_BE_NULL);
+    } else {
+      switch (field) {
+        case "mappingKey" -> builder.mappingKey();
```

Comment:
❓ Sorry I didn't pick this up before, with other indexes we've had an issue in searching in that the search request expects there to be a `key` field, I believe its used as a default sorting etc. Can you confirm that if you run this locally and search the request passes?